easybuild/easyconfigs/b/Boost/Boost-1.47.0-goolf-1.4.10-Python-2.7.5.eb

```diff
@@ -0,0 +1,26 @@
+name = 'Boost'
+version = '1.47.0'
+
+homepage = 'http://www.boost.org/'
+description = """Boost provides free peer-reviewed portable C++ source libraries."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%%(namelower)s_%s.tar.gz' % '_'.join(version.split('.'))]
+
+pythonversion = '2.7.5'
+versionsuffix = '-Python-%s' % pythonversion
+
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('Python', pythonversion),
+]
+
+# also build boost_mpi
+boost_mpi = True
+
+osdependencies = ['zlib-devel']
```

Comment:
@boegel: my guess is that Pablo added this line here, in order to get a sane warning before things go astray;
this kind of need falls into the bigger discussion of generic dependencies management (left-over from 1st hackathon!).